.github/workflows/PR.yml

```diff
@@ -109,6 +109,7 @@ jobs:
           # Clean up lingering build artifacts
           for n in linux-64 osx-64 noarch; do
               rm -f /opt/mambaforge/envs/bioconda/conda-bld/$n/*.tar.bz2
+              rm -f /opt/mambaforge/envs/bioconda/conda-bld/$n/*.conda
```

Comment:
N.B.: The workflow(s) needs an overhaul generally so we can remove these
undesired parts.
Meaning, we should either remove the caching overall or at least be more
precise with what's being cached (e.g., separate bioconda-utils'
environment via "pkgs_dirs" and "conda_build.root-dir" .condarc config).